pkg/controller/provider/container/vsphere/model.go

```diff
@@ -1152,6 +1184,7 @@ func (v *VmAdapter) updateDisks(devArray *types.ArrayOfVirtualDevice) {
 					Bus:            bus,
 					Serial:         backing.Uuid,
 					WinDriveLetter: winDriveLetter,
+					ChangeTrackingEnabled: backing.ChangeId != "",
```

Comment:
I don't like indirectly assuming the state of ctkEnabled from the existence of a change ID.